packages/builder/src/components/automation/AutomationPanel/AutomationList.svelte

```diff
@@ -23,7 +24,7 @@
 </script>
 
 <div class="automations-list">
-  {#each $automationStore.automations as automation, idx}
+  {#each _.orderBy($automationStore.automations, ["name"], ["asc"]) as automation, idx}
```

Comment:
Can be done without lodash and simplified to:
`$automationStore.automations.sort(aut => aut.name)`